### PR TITLE
Changing the tetris results page and the randomization model

### PR DESCRIPTION
**Pull request by johnjelliott**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -133,67 +133,87 @@ def base_income_assignment(self):
...
+
+            
+
+#        shuffled_ids_3 = shuffled_group_ids[:3]
+#        shuffled_ids_6 = shuffled_group_ids[:6]
+        
 
-        print(shuffled_ids_3)
-        print(shuffled_ids_6)
+#        for p in self.get_players():
```
Comment (mgutierrezc) [inline models.py:159]: You can erase this code if it's not required. Github keeps track of it

File: models.py [hunk 2]
```diff
@@ -133,67 +133,87 @@ def base_income_assignment(self):
...
-                    p.shuffled = True
-                    j6 += 1
-                elif p.id_in_group not in shuffled_ids_6:
-                    p.base_earnings = p.real_effort_earnings
-                    p.shuffled = False
-                else:
-                    print("Error: No 'player.id_in_group' value for assignment of shuffled earning")
-            else:
-                print("Error: No 'luck' value for assignment of shuffled earning")
+#        for p in self.get_players():
```
Comment (mgutierrezc) [inline models.py:193]: same here

File: models.py [hunk 3]
```diff
@@ -133,67 +133,87 @@ def base_income_assignment(self):
             shuffled_group_ids.append(p.id_in_group)
         random.SystemRandom().shuffle(shuffled_group_ids)
 
-        shuffled_ids_3 = shuffled_group_ids[:3]
-        shuffled_ids_6 = shuffled_group_ids[:6]
+        dexrand = 0
```
Comment (mgutierrezc) [inline models.py:136]: Try adding a short and single line comment before your `dex` vars to explain how these work

**Comment by johnjelliott** [comment]
I changed the comments

**Review by mgutierrezc — APPROVED** [review]

**Pull request merged**